docroot/modules/custom/vies_application/src/EDBBrugsenIntegrationVies.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Drupal\vies_application;
+
+/**
+ * @file
+ * Contains \Drupal\vih_subscription\Misc\EDBBrugsenIntegration.
+ */
+
+use Drupal\vih_subscription\Misc\EDBBrugsenIntegration;
+
+/**
+ * VIES EDBBrugsen service integration class.
+ *
+ * Why do we need this? Because when the token API was moved to core we did not
+ * reuse the entity type as the base name for taxonomy terms and vocabulary
+ * tokens.
+ */
+class EDBBrugsenIntegrationVies extends EDBBrugsenIntegration {
```

Comment:
in my opinion, the function below can be just added to original "EDBBrugsenIntegration", it does not provide any changes to existing functionality, only adding new.

Is there any particular reason for introducing VIES specific way of EDBBrugsen Integration?